scripts/i.spectral/i.spectral.html

```diff
@@ -30,10 +30,12 @@ <h2>EXAMPLE</h2>
 <div class="code"><pre>
 g.region raster=lsat7_2002_10 -p
 LIST=`g.list type=raster pattern="lsat7_2002_[1-5,7]0" separator=","`
+d.mon start=cairo output=plot.png
 i.spectral input=$LIST coordinates=637502.25,221744.25
+d.mon stop=cairo
 </pre></div>
 
-This will search all LANDSAT map for 2002 but select only the B, G, R, 
+This will search all LANDSAT map for 2002 but select only the B, G, R,
 NIR, and MIR channels.
 
 
```

Comment:
Please consider to add under "SEE ALSO" references to

```html
<a href="cairodriver.html">Cairo driver</a>,
<a href="pngdriver.html">PNG driver</a>,
<a href="psdriver.html">PS driver (Postscript)</a>
```

(and <h2>AUTHOR</h2> may become <h2>AUTHORS</h2>)

Reply:
All right, I fixed it.